src/actions/breakpoints.js

```diff
@@ -191,6 +191,34 @@ export function toggleAllBreakpoints(shouldDisableBreakpoints: boolean) {
   };
 }
 
+export function remapBreakpoints(sourceId: string) {
+  return async ({ dispatch, getState, sourceMaps }: ThunkArgs) => {
+    const breakpoints = getBreakpoints(getState());
+    const newBreakpoints = await remapLocations(
+      breakpoints,
+      sourceId,
+      sourceMaps
+    );
+
+    return dispatch({
+      type: "REMAP_BREAKPOINTS",
+      breakpoints: newBreakpoints
+    });
+  };
+}
+
+function remapLocations(breakpoints, sourceId, sourceMaps) {
```

Comment:
The promise api is still useful in async await, i just never had to use it before, and it was a stumbling block. Async/Await can resolve single promises, but not a collection. infos --> https://github.com/tc39/ecmascript-asyncawait/issues/61